frontend/express/libs/members.js

```diff
@@ -364,7 +364,7 @@ membersUtility.updateStats = function(member) {
                 timeout: 4E3,
```

Comment:
Check on https://github.com/Countly/countly-server/pull/4460/files#diff-a402ac28f1f8ba694491fae7b927df7679142198ce79470e986fba04ae8260b9R349 should include offline check and frontend.country_tracking check, no need for config check anymore